features/step_definitions/division_steps.rb

```diff
@@ -0,0 +1,52 @@
+require 'calculator'
+
+When /^I divide two numbers$/ do
+	@quotient = Calculator.new.divide 8,4
+ end
+
+Then /^I get the quotient$/ do
+    expect(@quotient).to eq 2
+ end
+  
+When /^I divide the same numbers in different orders$/ do
+    calc = Calculator.new
+	@quotient1 = calc.divide 8,3
+	@quotient2 = calc.divide 3,8
+ end
+
+Then /^the quotients are not the same$/ do
+	expect(@quotient1).not_to eq @quotient2
+ end
+  
+When /^I divide a number by one$/ do
+	@quotient = Calculator.new.divide 410,1
+ end
+
+Then /^the quotient should be the number$/ do
+	expect(@quotient).to eq 410
+ end
+ 
+When /^I divide zero by a integer$/ do
+	@quotient = Calculator.new.divide 0,4
+ end
+
+Then /^the quotient is zero$/ do
+	expect(@quotient).to eq 0
+ end
+
+ When /^I divide a number by a zero$/ do

```

Comment:
You didn't actually use these steps in your `division.feature` file. It would have worked, so it looks like it was just an oversight.